### ISSUE TITLE
L2 Error of ~0.02 After 100k Iterations with RAD Code for Allen-Cahn Equation

### ISSUE THREAD

**Issue by 320256** [lu-group/pinn-sampling#9]
I’ve been using your RAD code to solve the Allen-Cahn equation, but after ~100k iterations, the L2 error stabilizes around ​0.02, which is much higher than the results in your paper. Could you kindly share any implementation-specific suggestions to improve accuracy? 
Here are my current settings:
# Training configuration  
NumDomain = 500   
model.train(epochs=2000)  

No other parameters were modified from the original code (e.g., loss weights, optimization settings), but the accuracy remains suboptimal.I would greatly appreciate any suggestions to align my implementation with your results. Thank you for your time and expertise!
